src/main.zig

```diff
@@ -4761,6 +4764,23 @@ fn cmdInit(gpa: Allocator, arena: Allocator, args: []const []const u8) !void {
     return cleanExit();
 }
 
+fn sanitizeExampleName(arena: Allocator, bytes: []const u8) error{OutOfMemory}![]const u8 {
+    if (bytes.len == 0) return "foo";
+    var result: std.ArrayListUnmanaged(u8) = .empty;
+    try result.append(arena, switch (bytes[0]) {
+        '_', 'a'...'z', 'A'...'Z' => |c| c,
+        else => '_',
+    });
+    for (bytes[1..]) |byte| switch (byte) {
+        '_', 'a'...'z', 'A'...'Z', '0'...'9' => try result.append(arena, byte),
```

Comment:
```suggestion
        '_', 'a'...'z', 'A'...'Z', '0'...'9' => try result.append(arena, byte),
        '-', '.' => '_',
```
`-` and `.` are somewhat common in repo/project names, so substituting them with `_` instead of ignoring them probably gets you closer to what the user actually wants.